frontend/pages/SoftwarePage/SoftwareAddPage/SoftwareFleetMaintained/FleetMaintainedAppDetailsPage/FleetAppDetailsForm/FleetAppDetailsForm.tsx

```diff
@@ -48,8 +54,94 @@ interface IFleetAppDetailsFormProps {
   onSubmit: (formData: IFleetMaintainedAppFormData) => void;
 }
 
+interface IInstallTypeSection {
+  installType: InstallType;
+  onChangeInstallType: (value: string) => void;
+  isCustomPackage?: boolean;
+  isExeCustomPackage?: boolean;
+}
+
+// Also used in custom package form (PackageForm.tsx)
+export const InstallTypeSection = ({
```

Comment:
Looks good but if this also used in Package form I think it should live in that file and be imported here instead of the other way around. Seems strange to have `components/PackageForm` import a component from a specific page.

Reply:
@ghernandez345 PackageForm.tsx is the custom package page's version of FMA's FleetAppDetailsForm.tsx, I was thinking of making a components directory for stuff that is used in both but since it was only this item, I just left it where it was, WDYT?